### PR TITLE
Modern theme support for stackblitz demos

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**

### CODE REVIEW

File: src/app/public/modules/code-examples/code-example.component.ts [hunk 1]
```diff
@@ -23,17 +23,23 @@ import {
...
   @Input()
-  public sourceCodePath: string;
+  public heading: string;
 
   /**
-   * The heading to be used for the code example.
+   * Specifies if the editor service should show the example in modern theme.
+   */
+  @Input()
+  public modernTheme: boolean = false;
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/code-examples/code-example.component.ts:36]: To make this future-proof, I'd use an enum:
```
enum SkyDocsCodeExampleTheme { Default, Modern }
```

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-modern-stackblitz](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-modern-stackblitz)

File: src/app/public/modules/code-examples/code-example-theme.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export enum SkyDocsCodeExampleTheme {
+  Default,
+  Modern
```
Comment (Blackbaud-SteveBrush) [inline src/app/public/modules/code-examples/code-example-theme.ts:3]: If you used `string` enum, could you pass the value into the input as a string?
```
export enum SkyDocsCodeExampleTheme {
  Default = 'default',
  Modern = 'modern'
}
```
In the template...
```
<sky-docs-code-example theme="modern">
```

File: src/app/docs/sample/sample.component.ts [hunk 3]
```diff
@@ -3,6 +3,10 @@ import {
   Component
 } from '@angular/core';
 
+import {
+  SkyDocsCodeExampleTheme
+} from '../../public/modules/code-examples/code-example-theme';
```
Comment (Blackbaud-SteveBrush) [inline src/app/docs/sample/sample.component.ts:8]: Should this be pulled from `../../public/public_api`?

File: src/app/public/modules/code-examples/code-example-theme.ts (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/code-examples/code-example-theme.ts]: Done.

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-modern-stackblitz](https://host.nxt.blackbaud.com/skyux-docs-tools?_sp.spa=pr-modern-stackblitz)

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**